pkg/objstore/s3/s3.go

```diff
@@ -81,6 +87,12 @@ func parseConfig(conf []byte) (Config, error) {
 	if config.PutUserMetadata == nil {
 		config.PutUserMetadata = make(map[string]string)
 	}
+
+	// Use the default minPartSize if not set.
```

Comment:
This comment might be not necessary, but not a blocker.

Reply:
fixed